AEPMessaging.xcodeproj/project.pbxproj

```diff
@@ -2193,13 +2192,12 @@
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
 				INFOPLIST_FILE = AEPMessaging/Sources/Info.plist;
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
-				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
```

Comment:
Shouldn't it have also added `IPHONEOS_DEPLOYMENT_TARGET = 11.0;`

Reply:
11.0 is the project setting.  10.0 was a target override.  the remove of this line causes the target to use the value set in the project (11.0)